### ISSUE TITLE
Shift count overflow: R3BCalifaFebexReader

### ISSUE THREAD

**Issue by kresan** [R3BRootGroup/R3BRoot#175]
At line 63 in r3bsource/R3BCalifaFebexReader.cxx:
bitwise shift count of 32 is equal to size of uint32_t:
```
uint64_t timestamp = ((uint32_t) fData->CALIFA_TSMSBv[crystal] << 32)
      | (uint32_t) fData->CALIFA_TSLSBv[crystal];
```
Should we use uint64_t in the casts?

Cheers,
Dima

**Comment by hapol** [comment]
Yes, both casts  after the = should be equal to uint64.
Preparing the PR...
Cheers,
Héctor

**Comment by kresan** [comment]
Solved in #181 
Thanks Héctor!
